openedx/core/djangoapps/auth_exchange/tests/utils.py

```diff
@@ -2,10 +2,16 @@
 Test utilities for OAuth access token exchange
 """
 
-from social.apps.django_app.default.models import UserSocialAuth
+from django.conf import settings
+from social_django.models import UserSocialAuth, Partial
+
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixin
 
 
+TPA_FEATURES_KEY = 'ENABLE_THIRD_PARTY_AUTH'
+TPA_FEATURE_ENABLED = TPA_FEATURES_KEY in settings.FEATURES
```

Comment:
These seem to duplicate `testutil.AUTH_FEATURE_ENABLED` and `testutil.AUTH_FEATURES_KEY` in `third_party_auth`, no?